From: Debian Cyrus Team
 <pkg-cyrus-imapd-debian-devel@lists.alioth.debian.org>
Date: Tue, 4 Aug 2015 10:38:04 +0200
Subject: 05-fix_programnames.dpatch

#! /bin/sh /usr/share/dpatch/dpatch-run
## 05-fix_programnames_in_manpages.dpatch by Sven Mueller <debian@incase.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: fix some minor things in manpages, fix program names in manpages and
## DP: program sources

@DPATCH@
---
 imap/deliver.c     |  2 +-
 imap/quota.c       |  2 +-
 imap/reconstruct.c |  2 +-
 man/chk_cyrus.8    |  2 +-
 man/ctl_cyrusdb.8  |  2 +-
 man/ctl_deliver.8  |  2 +-
 man/ctl_mboxlist.8 |  2 +-
 man/cyr_expire.8   |  2 +-
 man/cyrus.conf.5   | 22 +++++++++++-----------
 man/deliver.8      | 12 ++++++------
 man/fud.8          |  2 +-
 man/idled.8        |  4 ++--
 man/imapd.8        |  4 ++--
 man/lmtpd.8        |  4 ++--
 man/master.8       | 20 ++++++++++----------
 man/nntpd.8        |  4 ++--
 man/notifyd.8      |  2 +-
 man/pop3d.8        |  4 ++--
 man/quota.8        | 14 +++++++-------
 man/reconstruct.8  | 38 +++++++++++++++++++-------------------
 man/smmapd.8       |  2 +-
 man/timsieved.8    |  4 ++--
 man/tls_prune.8    |  2 +-
 23 files changed, 77 insertions(+), 77 deletions(-)

diff --git a/imap/deliver.c b/imap/deliver.c
index 31ead1a..a60d0a3 100644
--- a/imap/deliver.c
+++ b/imap/deliver.c
@@ -113,7 +113,7 @@ static struct backend *init_net(const char *sockaddr);
 static void usage(void)
 {
     fprintf(stderr,
-            "421-4.3.0 usage: deliver [-C <alt_config> ] [-m mailbox]"
+            "421-4.3.0 usage: cyrdeliver [-C <alt_config> ] [-m mailbox]"
             " [-a auth] [-r return_path] [-l] [-D]\r\n");
     fprintf(stderr, "421 4.3.0 %s\n", cyrus_version());
     exit(EC_USAGE);
diff --git a/imap/quota.c b/imap/quota.c
index e55e0bf..116d084 100644
--- a/imap/quota.c
+++ b/imap/quota.c
@@ -227,7 +227,7 @@ int main(int argc,char **argv)
 static void usage(void)
 {
     fprintf(stderr,
-            "usage: quota [-C <alt_config>] [-d <domain>] [-f] [-q] [prefix]...\n");
+            "usage: cyrquota [-C <alt_config>] [-d <domain>] [-f] [-q] [prefix]...\n");
     exit(EC_USAGE);
 }
 
diff --git a/imap/reconstruct.c b/imap/reconstruct.c
index c896b32..590872e 100644
--- a/imap/reconstruct.c
+++ b/imap/reconstruct.c
@@ -421,7 +421,7 @@ int main(int argc, char **argv)
 static void usage(void)
 {
     fprintf(stderr,
-            "usage: reconstruct [-C <alt_config>] [-p partition] [-ksrfxu] mailbox...\n");
+            "usage: cyrreconstruct [-C <alt_config>] [-p partition] [-ksrfxu] mailbox...\n");
     fprintf(stderr, "       reconstruct [-C <alt_config>] -m\n");
     exit(EC_USAGE);
 }
diff --git a/man/chk_cyrus.8 b/man/chk_cyrus.8
index 940f190..439759e 100644
--- a/man/chk_cyrus.8
+++ b/man/chk_cyrus.8
@@ -129,7 +129,7 @@ Perform consistancy checks on mailbox \fIuser.marysmith\fP\&.
 .UNINDENT
 .SH SEE ALSO
 .sp
-imapd.conf(5), master(8)
+imapd.conf(5), cyrmaster(8)
 .SH AUTHOR
 The Cyrus Team, Nic Bernstein (Onlight), Jeroen van Meeuwen (Kolab Systems)
 .SH COPYRIGHT
diff --git a/man/ctl_cyrusdb.8 b/man/ctl_cyrusdb.8
index 288e00b..aa87013 100644
--- a/man/ctl_cyrusdb.8
+++ b/man/ctl_cyrusdb.8
@@ -142,7 +142,7 @@ cyrus.conf(5)\&.
 /etc/cyrus.conf
 .SH SEE ALSO
 .sp
-cyrus.conf(5), imapd.conf(5), master(8)
+cyrus.conf(5), imapd.conf(5), cyrmaster(8)
 .SH AUTHOR
 The Cyrus Team, Nic Bernstein (Onlight), Jeroen van Meeuwen (Kolab Systems)
 .SH COPYRIGHT
diff --git a/man/ctl_deliver.8 b/man/ctl_deliver.8
index e1f4ff1..5725f15 100644
--- a/man/ctl_deliver.8
+++ b/man/ctl_deliver.8
@@ -110,7 +110,7 @@ Dump \fIconfigdirectory\fP/\fBdeliver.db\fP to \fI/tmp/deliverdb.txt\fP\&.
 .UNINDENT
 .SH SEE ALSO
 .sp
-imapd.conf(5), master(8)
+imapd.conf(5), cyrmaster(8)
 .SH FILES
 .sp
 /etc/imapd.conf,
diff --git a/man/ctl_mboxlist.8 b/man/ctl_mboxlist.8
index 80efc87..04238a6 100644
--- a/man/ctl_mboxlist.8
+++ b/man/ctl_mboxlist.8
@@ -238,7 +238,7 @@ asking for verification before deleting any local mailboxes.
 /etc/imapd.conf, <configdirectory>/mailboxes.db
 .SH SEE ALSO
 .sp
-imapd.conf(5), master(8)
+imapd.conf(5), cyrmaster(8)
 .SH AUTHOR
 The Cyrus Team, Nic Bernstein (Onlight), Jeroen van Meeuwen (Kolab Systems)
 .SH COPYRIGHT
diff --git a/man/cyr_expire.8 b/man/cyr_expire.8
index 6d89bf6..6bbe558 100644
--- a/man/cyr_expire.8
+++ b/man/cyr_expire.8
@@ -236,7 +236,7 @@ version 3.0.
 /etc/imapd.conf
 .SH SEE ALSO
 .sp
-imapd.conf(5), master(8), cyradm(8)
+imapd.conf(5), cyrmaster(8), cyradm(8)
 .SH AUTHOR
 The Cyrus Team, Nic Bernstein (Onlight), Jeroen van Meeuwen (Kolab Systems)
 .SH COPYRIGHT
diff --git a/man/cyrus.conf.5 b/man/cyrus.conf.5
index 54d760b..c48179e 100644
--- a/man/cyrus.conf.5
+++ b/man/cyrus.conf.5
@@ -62,9 +62,9 @@ Cyrus configuration file
 .SH DESCRIPTION
 .sp
 \fBcyrus.conf\fP is the configuration file for the Cyrus
-master(8) process.  It defines the startup procedures,
+cyrmaster(8) process.  It defines the startup procedures,
 services, events and daemons to be spawned, managed and tended to by
-\fBmaster\fP\&.
+\fBcyrmaster\fP\&.
 .sp
 The \fB/etc/cyrus.conf\fP file consists of a series of entries divided
 into sections of the form
@@ -97,19 +97,19 @@ file.  The arguments that are available for each entry within the
 section are described, and each argument\(aqs default value is shown.
 .sp
 An important distinction exists between \fBSERVICES\fP and \fBDAEMON\fP ;
-the former have sockets which master(8) will listen on
+the former have sockets which cyrmaster(8) will listen on
 (either IP or Unix domain) while the latter do not.  Similarly,
 processes listed in \fBSTART\fP will be run to completion before any
 \fBSERVICES\fP are started, while those in \fBDAEMON\fP will be managed by
-master(8)\&.
+cyrmaster(8)\&.
 .sp
 \fBNOTE:\fP
 .INDENT 0.0
 .INDENT 3.5
-If master(8) is started in debugging mode (\fB\-D\fP) the
-behavior of \fBDAEMON\fP will be altered, as master(8)
+If cyrmaster(8) is started in debugging mode (\fB\-D\fP) the
+behavior of \fBDAEMON\fP will be altered, as cyrmaster(8)
 will no longer be backgrounded.  Thus, processes started under
-DAEMON may not be terminated by master(8)\&.
+DAEMON may not be terminated by cyrmaster(8)\&.
 .UNINDENT
 .UNINDENT
 .sp
@@ -139,7 +139,7 @@ string argument is required.
 Prior to v3, non\-service daemons like \fBidled\fP were started from
 \fBSTART\fP but would background themselves, thus not blocking.  Post
 v3 these are better managed through the \fBDAEMON\fP section, under
-which master will provide life\-cycle management (i.e. restarting
+which cyrmaster will provide life\-cycle management (i.e. restarting
 dead processes).
 .UNINDENT
 .UNINDENT
@@ -254,7 +254,7 @@ process. This integer value is optional.
 .fi
 .INDENT 0.0
 .INDENT 3.5
-Maximum number of processes to fork per second \- the master
+Maximum number of processes to fork per second \- the cyrmaster
 will insert sleeps to ensure it doesn\(aqt fork faster than this
 on average.
 .UNINDENT
@@ -300,7 +300,7 @@ set to 1440. This string argument is optional.
 .sp
 This section lists long running daemons to start before any
 \fBSERVICES\fP are spawned.  They will be shutdown when
-master(8) is exiting.
+cyrmaster(8) is exiting.
 .sp
 .nf
 \fBcmd=\fP<no default>
@@ -348,7 +348,7 @@ different modes or configurations (i.e., SSL and non\-SSL enabled) yet
 still have separate access control rules.
 .SH SEE ALSO
 .sp
-master(8),
+cyrmaster(8),
 imapd(8),
 pop3d(8),
 lmtpd(8),
diff --git a/man/deliver.8 b/man/deliver.8
index ff344d3..8ace8d2 100644
--- a/man/deliver.8
+++ b/man/deliver.8
@@ -2,7 +2,7 @@
 .
 .TH "DELIVER" "8" "August 10, 2018" "3.0.8" "Cyrus IMAP"
 .SH NAME
-deliver \- Cyrus IMAP documentation
+cyrdeliver \- Cyrus IMAP documentation
 .
 .nr rst2man-indent-level 0
 .
@@ -62,17 +62,17 @@ intro
 .SH SYNOPSIS
 .sp
 .nf
-\fBdeliver\fP [ \fB\-C\fP \fIconfig\-file\fP ] [ \fB\-d\fP ] [ \fB\-r\fP \fIaddress\fP ]
+\fBcyrdeliver\fP [ \fB\-C\fP \fIconfig\-file\fP ] [ \fB\-d\fP ] [ \fB\-r\fP \fIaddress\fP ]
        [ \fB\-f\fP \fIaddress\fP ] [ \fB\-m\fP \fImailbox\fP ] [ \fB\-a\fP \fIauth\-id\fP ]
        [ \fB\-q\fP ] [ \fIuserid\fP ]...
-\fBdeliver\fP [ \fB\-C\fP \fIconfig\-file\fP ] \fB\-l\fP
+\fBcyrdeliver\fP [ \fB\-C\fP \fIconfig\-file\fP ] \fB\-l\fP
 .fi
 .SH DESCRIPTION
 .sp
-\fBdeliver\fP reads a message from the standard input and delivers it to
+\fBcyrdeliver\fP reads a message from the standard input and delivers it to
 one or more IMAP mailboxes.
 .sp
-\fBdeliver\fP reads its configuration options out of the imapd.conf(5) file unless specified otherwise by \fB\-C\fP\&.
+\fBcyrdeliver\fP reads its configuration options out of the imapd.conf(5) file unless specified otherwise by \fB\-C\fP\&.
 .SH OPTIONS
 .INDENT 0.0
 .TP
@@ -123,7 +123,7 @@ Accept messages using the LMTP protocol.
 .UNINDENT
 .SH NOTES
 .sp
-Depending on the setting of \fBreject8bit\fP in imapd.conf(5), deliver
+Depending on the setting of \fBreject8bit\fP in imapd.conf(5), cyrdeliver
 either rejects/accepts messages with 8\-bit\-set characters in the headers.
 If we accept messages with 8\-bit\-set characters in the headers, then depending
 on the setting of \fBmunge8bit\fP, these characters are either left un\-touched
diff --git a/man/fud.8 b/man/fud.8
index 969f5d2..1635e1a 100644
--- a/man/fud.8
+++ b/man/fud.8
@@ -66,7 +66,7 @@ Provide information about user mailboxes
 .fi
 .SH DESCRIPTION
 .sp
-\fBfud\fP is a long lived datagram daemon started from master that
+\fBfud\fP is a long lived datagram daemon started from cyrmaster that
 provides information about when a user last read their mail, when mail
 last arrived in a user\(aqs mailbox, and how many messages are recent for
 that user.
diff --git a/man/idled.8 b/man/idled.8
index 143fa0e..8264b93 100644
--- a/man/idled.8
+++ b/man/idled.8
@@ -70,7 +70,7 @@ Provide near real\-time updates for IMAP IDLE
 mailbox changes and signals the appropriate \fBimapd\fP to report the
 changes to the client.
 .sp
-\fBIdled\fP is usually started from master(8)\&.
+\fBIdled\fP is usually started from cyrmaster(8)\&.
 .sp
 \fBidled\fP reads its configuration options out of the imapd.conf(5) file unless specified otherwise by \fB\-C\fP\&.
 .SH OPTIONS
@@ -96,7 +96,7 @@ DAEMON {
 .sp
 imapd.conf(5),
 cyrus.conf(5),
-master(8)
+cyrmaster(8)
 .SH AUTHOR
 The Cyrus Team, Nic Bernstein (Onlight)
 .SH COPYRIGHT
diff --git a/man/imapd.8 b/man/imapd.8
index 53b7ba8..b8b4688 100644
--- a/man/imapd.8
+++ b/man/imapd.8
@@ -69,7 +69,7 @@ IMAP server process
 .sp
 \fBimapd\fP is an IMAP4rev1 server.  It accepts commands on its standard
 input and responds on its standard output.  It MUST be invoked by
-master(8) with those descriptors attached to a remote client
+cyrmaster(8) with those descriptors attached to a remote client
 connection.
 .sp
 \fBimapd\fP reads its configuration options out of the imapd.conf(5) file unless specified otherwise by \fB\-C\fP\&.
@@ -159,7 +159,7 @@ SERVICES {
 .SH SEE ALSO
 .sp
 imapd.conf(5),
-master(8)
+cyrmaster(8)
 .SH AUTHOR
 The Cyrus Team, Nic Bernstein (Onlight)
 .SH COPYRIGHT
diff --git a/man/lmtpd.8 b/man/lmtpd.8
index 42a6a9d..ee97497 100644
--- a/man/lmtpd.8
+++ b/man/lmtpd.8
@@ -68,7 +68,7 @@ LMTP server process
 .sp
 \fBlmtpd\fP is an LMTP server used to deliver mail to the IMAP mailstore.
 It accepts commands on its standard input and responds on its standard
-output. It MUST be invoked by master(8) with those
+output. It MUST be invoked by cyrmaster(8) with those
 descriptors attached to a remote client connection.
 .sp
 \fBlmtpd\fP reads its configuration options out of the imapd.conf(5) file unless specified otherwise by \fB\-C\fP\&.
@@ -124,7 +124,7 @@ SERVICES {
 .SH SEE ALSO
 .sp
 imapd.conf(5),
-master(8)
+cyrmaster(8)
 .SH AUTHOR
 The Cyrus Team, Nic Bernstein (Onlight)
 .SH COPYRIGHT
diff --git a/man/master.8 b/man/master.8
index cecb41c..8646f54 100644
--- a/man/master.8
+++ b/man/master.8
@@ -2,7 +2,7 @@
 .
 .TH "MASTER" "8" "August 10, 2018" "3.0.8" "Cyrus IMAP"
 .SH NAME
-master \- Cyrus IMAP documentation
+cyrmaster \- Cyrus IMAP documentation
 .
 .nr rst2man-indent-level 0
 .
@@ -62,14 +62,14 @@ The Cyrus IMAP master process.
 .SH SYNOPSIS
 .sp
 .nf
-\fBmaster\fP [ \fB\-C\fP \fIconfig\-file\fP ] [ \fB\-M\fP \fIalternate cyrus.conf\fP ]
+\fBcyrmaster\fP [ \fB\-C\fP \fIconfig\-file\fP ] [ \fB\-M\fP \fIalternate cyrus.conf\fP ]
     [ \fB\-l\fP \fIlisten queue\fP ] [ \fB\-p\fP \fIpidfile\fP ] [ \fB\-P\fP \fIsnmp agentx ping interval\fP ]
     [ \fB\-j\fP \fIjanitor period\fP ] [ \fB\-d\fP | \fB\-D\fP ] [ \fB\-L\fP \fIlogfile\fP ]
     [ \fB\-x\fP \fIsnmp agentx socket\fP ]
 .fi
 .SH DESCRIPTION
 .sp
-\fBmaster\fP is the process that controls all of the Cyrus
+\fBcyrmaster\fP is the process that controls all of the Cyrus
 processes. This process is responsible for creating all imapd, pop3d,
 lmtpd and sieved child processes. This process also initializes the
 Berkeley DB code and performs scheduled cleanup/maintenance.
@@ -78,7 +78,7 @@ If this process dies, then no new sessions will be started.
 .sp
 It kills itself and all child processes when it receives a SIGTERM.
 .sp
-\fBmaster\fP reads its configuration options out of the imapd.conf(5) file unless specified otherwise by \fB\-C\fP\&.
+\fBcyrmaster\fP reads its configuration options out of the imapd.conf(5) file unless specified otherwise by \fB\-C\fP\&.
 .SH OPTIONS
 .INDENT 0.0
 .TP
@@ -88,7 +88,7 @@ Use the specified configuration file \fIconfig\-file\fP rather than the default
 .INDENT 0.0
 .TP
 .B \-M  alternate cyrus.conf
-Specifies an alternate cyrus.conf for use by master.
+Specifies an alternate cyrus.conf for use by cyrmaster.
 .UNINDENT
 .INDENT 0.0
 .TP
@@ -111,7 +111,7 @@ entries).
 .TP
 .B \-p  pidfile
 Use \fIpidfile\fP as the pidfile.  If not specified, defaults to
-\fB/var/run/master.pid\fP
+\fB/var/run/cyrmaster.pid\fP
 .UNINDENT
 .INDENT 0.0
 .TP
@@ -146,11 +146,11 @@ Requires net\-snmp 5.0 or higher.
 .UNINDENT
 .SH CONFIGURATION
 .sp
-Upon execution, \fBmaster\fP reads its configuration information
+Upon execution, \fBcyrmaster\fP reads its configuration information
 out of the cyrus.conf(5) file, or an alternative if \fB\-M\fP
 is used.
 .sp
-\fBmaster\fP rereads its configuration file when it receives a
+\fBcyrmaster\fP rereads its configuration file when it receives a
 hangup signal, SIGHUP.  Services and events may be added, deleted or
 modified when the configuration file is reread.  Any active services
 removed from the configuration file will be allowed to run until
@@ -158,7 +158,7 @@ completion.  Services added or modified to listen on a privileged port
 may not be able to bind the port, depending on your system
 configuration.  In this case a full restart is needed.
 .sp
-\fBmaster\fP propagates the hangup signal, SIGHUP, to its child
+\fBcyrmaster\fP propagates the hangup signal, SIGHUP, to its child
 service processes, so that they stop waiting for incoming connections
 and exit, allowing them to be recycled.  This is useful to make
 services take into account a new version of the
@@ -176,7 +176,7 @@ Setting it higher results in more log messages being generated.
 .sp
 /etc/cyrus.conf,
 /etc/imapd.conf,
-/var/run/master.pid
+/var/run/cyrmaster.pid
 .SH SEE ALSO
 .sp
 cyrus.conf(5), imapd.conf(5), imapd(8),
diff --git a/man/nntpd.8 b/man/nntpd.8
index 8cfc89d..1488f0d 100644
--- a/man/nntpd.8
+++ b/man/nntpd.8
@@ -69,7 +69,7 @@ NNTP server process
 .sp
 \fBnntpd\fP is an NNTP server. It accepts commands on its standard input
 and responds on its standard output. It MUST invoked by
-master(8) with those descriptors attached to a remote client
+cyrmaster(8) with those descriptors attached to a remote client
 connection.
 .sp
 \fBnntpd\fP reads its configuration options out of the imapd.conf(5) file unless specified otherwise by \fB\-C\fP\&.  The optional \fBnewsprefix\fP setting
@@ -148,7 +148,7 @@ Any higher SSF implies some form of privacy protection.
 .SH SEE ALSO
 .sp
 imapd.conf(5),
-master(8)
+cyrmaster(8)
 .SH AUTHOR
 The Cyrus Team, Nic Bernstein (Onlight)
 .SH COPYRIGHT
diff --git a/man/notifyd.8 b/man/notifyd.8
index c0ea2ab..5ef6d7a 100644
--- a/man/notifyd.8
+++ b/man/notifyd.8
@@ -66,7 +66,7 @@ Cyrus notification server
 .fi
 .SH DESCRIPTION
 .sp
-\fBnotifyd\fP is a daemon started from master(8) that handles
+\fBnotifyd\fP is a daemon started from cyrmaster(8) that handles
 notification requests on behalf of lmtpd(8)\&. \fBNotifyd\fP
 accepts the request and performs the notification using the method
 specified in the request.
diff --git a/man/pop3d.8 b/man/pop3d.8
index acf834c..b6d0b94 100644
--- a/man/pop3d.8
+++ b/man/pop3d.8
@@ -69,7 +69,7 @@ POP3 server process
 .sp
 \fBpop3d\fP is an POP3 server.  It accepts commands on its standard
 input and responds on its standard output.  It MUST be invoked by
-master(8) with those descriptors attached to a remote client
+cyrmaster(8) with those descriptors attached to a remote client
 connection.
 .sp
 \fBpop3d\fP reads its configuration options out of the imapd.conf(5) file unless specified otherwise by \fB\-C\fP\&.
@@ -145,7 +145,7 @@ SERVICES {
 .SH SEE ALSO
 .sp
 imapd.conf(5),
-master(8)
+cyrmaster(8)
 .SH AUTHOR
 The Cyrus Team, Nic Bernstein (Onlight)
 .SH COPYRIGHT
diff --git a/man/quota.8 b/man/quota.8
index 0edb422..080b978 100644
--- a/man/quota.8
+++ b/man/quota.8
@@ -2,7 +2,7 @@
 .
 .TH "QUOTA" "8" "August 10, 2018" "3.0.8" "Cyrus IMAP"
 .SH NAME
-quota \- Cyrus IMAP documentation
+cyrquota \- Cyrus IMAP documentation
 .
 .nr rst2man-indent-level 0
 .
@@ -62,14 +62,14 @@ Report and optionally fix storage and message quota usage.
 .SH SYNOPSIS
 .sp
 .nf
-\fBquota\fP [ \fB\-C\fP \fIconfig\-file\fP ] [ \fB\-d\fP \fIdomain\fP ] [ \fB\-f\fP ] [ \fImailbox\-prefix\fP\&... ]
+\fBcyrquota\fP [ \fB\-C\fP \fIconfig\-file\fP ] [ \fB\-d\fP \fIdomain\fP ] [ \fB\-f\fP ] [ \fImailbox\-prefix\fP\&... ]
 .fi
 .SH DESCRIPTION
 .sp
-\fBquota\fP generates a report listing quota roots, giving their limits and
+\fBcyrquota\fP generates a report listing quota roots, giving their limits and
 usage.
 .sp
-If the \fB\-f\fP option is specified, \fBquota\fP first fixes any
+If the \fB\-f\fP option is specified, \fBcyrquota\fP first fixes any
 inconsistencies in the quota subsystem, such as mailboxes with the wrong
 quota root or quota roots with the wrong quota usage reported.
 .sp
@@ -84,12 +84,12 @@ that start with one of the given prefixes.
 \fBWARNING:\fP
 .INDENT 0.0
 .INDENT 3.5
-Running \fBquota\fP with both the \fB\-f\fP option and \fImailbox\-prefix\fP
+Running \fBcyrquota\fP with both the \fB\-f\fP option and \fImailbox\-prefix\fP
 arguments is not recommended.
 .UNINDENT
 .UNINDENT
 .sp
-\fBquota\fP reads its configuration options out of the imapd.conf(5) file unless specified otherwise by \fB\-C\fP\&.
+\fBcyrquota\fP reads its configuration options out of the imapd.conf(5) file unless specified otherwise by \fB\-C\fP\&.
 .SH OPTIONS
 .INDENT 0.0
 .TP
@@ -127,7 +127,7 @@ specified \fImailbox\-prefix\fP\&.
 .SH EXAMPLES
 .sp
 .nf
-\fBquota\fP
+\fBcyrquota\fP
 .fi
 .INDENT 0.0
 .INDENT 3.5
diff --git a/man/reconstruct.8 b/man/reconstruct.8
index a1f75e5..dc2c85b 100644
--- a/man/reconstruct.8
+++ b/man/reconstruct.8
@@ -2,7 +2,7 @@
 .
 .TH "RECONSTRUCT" "8" "August 10, 2018" "3.0.8" "Cyrus IMAP"
 .SH NAME
-reconstruct \- Cyrus IMAP documentation
+cyrreconstruct \- Cyrus IMAP documentation
 .
 .nr rst2man-indent-level 0
 .
@@ -62,44 +62,44 @@ Reconstruct mailboxes
 .SH SYNOPSIS
 .sp
 .nf
-\fBreconstruct\fP [ \fB\-C\fP \fIconfig\-file\fP ] [ \fB\-p\fP \fIpartition\fP ] [ \fB\-x\fP ] [ \fB\-r\fP ]
+\fBcyrreconstruct\fP [ \fB\-C\fP \fIconfig\-file\fP ] [ \fB\-p\fP \fIpartition\fP ] [ \fB\-x\fP ] [ \fB\-r\fP ]
     [ \fB\-f\fP ] [ \fB\-U\fP ] [ \fB\-s\fP ] [ \fB\-q\fP ] [ \fB\-G\fP ] [ \fB\-R\fP ] [ \fB\-o\fP ]
     [ \fB\-O\fP ] [ \fB\-M\fP ] \fImailbox\fP\&...
 
-\fBreconstruct\fP [ \fB\-C\fP \fIconfig\-file\fP ] [ \fB\-p\fP \fIpartition\fP ] [ \fB\-x\fP ] [ \fB\-r\fP ]
+\fBcyrreconstruct\fP [ \fB\-C\fP \fIconfig\-file\fP ] [ \fB\-p\fP \fIpartition\fP ] [ \fB\-x\fP ] [ \fB\-r\fP ]
     [ \fB\-f\fP ] [ \fB\-U\fP ] [ \fB\-s\fP ] [ \fB\-q\fP ] [ \fB\-G\fP ] [ \fB\-R\fP ] [ \fB\-o\fP ]
     [ \fB\-O\fP ] [ \fB\-M\fP ] [ \fB\-u\fP ] \fIusers\fP\&...
 
-\fBreconstruct\fP [ \fB\-C\fP \fIconfig\-file\fP ] [ \fB\-p\fP \fIpartition\fP ] [ \fB\-x\fP ] [ \fB\-r\fP ]
+\fBcyrreconstruct\fP [ \fB\-C\fP \fIconfig\-file\fP ] [ \fB\-p\fP \fIpartition\fP ] [ \fB\-x\fP ] [ \fB\-r\fP ]
     [ \fB\-f\fP ] [ \fB\-U\fP ] [ \fB\-s\fP ] [ \fB\-q\fP ] [ \fB\-G\fP ] [ \fB\-R\fP ] [ \fB\-o\fP ]
     [ \fB\-O\fP ] [ \fB\-M\fP ] \fB\-V\fP \fI<version>\fP [ \fB\-u\fP \fIusers\fP ]
 
-\fBreconstruct\fP [ \fB\-C\fP \fIconfig\-file\fP ] \fB\-m\fP
+\fBcyrreconstruct\fP [ \fB\-C\fP \fIconfig\-file\fP ] \fB\-m\fP
 .fi
 .SH DESCRIPTION
 .sp
-\fBreconstruct\fP rebuilds one or more IMAP mailboxes.  When invoked with
+\fBcyrreconstruct\fP rebuilds one or more IMAP mailboxes.  When invoked with
 the \fB\-m\fP switch, it rebuilds the master mailboxes file.  It can be
 used to recover from almost any sort of data corruption.
 .sp
-If \fBreconstruct\fP can find existing header and index files, it
+If \fBcyrreconstruct\fP can find existing header and index files, it
 attempts to preserve any data in them that is not derivable from the
-message files themselves. The state \fBreconstruct\fP attempts to
+message files themselves. The state \fBcyrreconstruct\fP attempts to
 preserve includes the flag names, flag state, and internaldate.
 .sp
-\fBreconstruct\fP derives all other information from the message files.
+\fBcyrreconstruct\fP derives all other information from the message files.
 .sp
-\fBreconstruct\fP reads its configuration options out of the imapd.conf(5) file unless specified otherwise by \fB\-C\fP\&.  Any mailbox directory underneath
+\fBcyrreconstruct\fP reads its configuration options out of the imapd.conf(5) file unless specified otherwise by \fB\-C\fP\&.  Any mailbox directory underneath
 the path specified in the \fBpartition\-news\fP configuration option is
 assumed to be in news format.
 .sp
-\fBreconstruct\fP does not adjust the quota usage recorded in any quota
-root files.  After running \fBreconstruct\fP, it is advisable to run
+\fBcyrreconstruct\fP does not adjust the quota usage recorded in any quota
+root files.  After running \fBcyrreconstruct\fP, it is advisable to run
 \fBquota(8)\fP with the \fB\-f\fP switch in order to fix the quota
 root files.
 .sp
 When upgrading versions of Cyrus software, it may be necessary to run
-\fBreconstruct\fP with the \fB\-V\fP option, to rebuild indexes to a
+\fBcyrreconstruct\fP with the \fB\-V\fP option, to rebuild indexes to a
 given version, (or \fImax\fP for the most recent).
 .SH OPTIONS
 .INDENT 0.0
@@ -139,9 +139,9 @@ restoring mailboxes from backups.
 .INDENT 0.0
 .TP
 .B \-s
-Don\(aqt stat underlying files.  This makes reconstruct run faster, at
+Don\(aqt stat underlying files.  This makes cyrreconstruct run faster, at
 the expense of not noticing some issues (like zero byte files or
-size mistmatches).  "\fBreconstruct \-s\fP" should be quite fast.
+size mistmatches).  "\fBcyrreconstruct \-s\fP" should be quite fast.
 .UNINDENT
 .INDENT 0.0
 .TP
@@ -164,7 +164,7 @@ individual message issues, including corrupted data files.
 .INDENT 0.0
 .TP
 .B \-I
-If two mailboxes exist with the same UNIQUEID and reconstruct visits
+If two mailboxes exist with the same UNIQUEID and cyrreconstruct visits
 both of them, \-I will cause the second mailbox to have a new UNIQUEID
 created for it.  If you don\(aqt specify \-I, you will just get a syslog
 entry telling you of the clash.
@@ -232,7 +232,7 @@ in the imapd.conf(5) file for additional mailboxes.
 .SH EXAMPLES
 .sp
 .nf
-\fBreconstruct \-r \-f\fP \fItech.support\fP
+\fBcyrreconstruct \-r \-f\fP \fItech.support\fP
 .fi
 .INDENT 0.0
 .INDENT 3.5
@@ -243,7 +243,7 @@ files.
 .UNINDENT
 .sp
 .nf
-\fBreconstruct \-r \-f\fP \fItech.support.Archive.2%\fP
+\fBcyrreconstruct \-r \-f\fP \fItech.support.Archive.2%\fP
 .fi
 .INDENT 0.0
 .INDENT 3.5
@@ -255,7 +255,7 @@ files.
 .UNINDENT
 .sp
 .nf
-\fBreconstruct \-r \-f \-u\fP \fIjsmith\fP
+\fBcyrreconstruct \-r \-f \-u\fP \fIjsmith\fP
 .fi
 .INDENT 0.0
 .INDENT 3.5
diff --git a/man/smmapd.8 b/man/smmapd.8
index 0259bc7..81f5826 100644
--- a/man/smmapd.8
+++ b/man/smmapd.8
@@ -69,7 +69,7 @@ Sendmail socket map daemon
 \fBsmmapd\fP is a Sendmail socket map daemon which is used to verify that
 a Cyrus mailbox exists, that it is postable and it is under quota.  It
 accepts commands on its standard input and responds on its standard
-output.  It MUST be invoked by master(8) with those
+output.  It MUST be invoked by cyrmaster(8) with those
 descriptors attached to a remote client connection.
 .sp
 \fBsmmapd\fP reads its configuration options out of the imapd.conf(5) file unless specified otherwise by \fB\-C\fP\&.
diff --git a/man/timsieved.8 b/man/timsieved.8
index de63a4a..ada1ecf 100644
--- a/man/timsieved.8
+++ b/man/timsieved.8
@@ -69,7 +69,7 @@ CMU hack for getting sieve scripts onto the server
 \fBtimsieved\fP is a server that allows users to remotely manage their
 sieve scripts kept on the server.  It accepts commands on its standard
 input and responds on its standard output. It MUST be invoked by
-master(8) with those descriptors attached to a remote client
+cyrmaster(8) with those descriptors attached to a remote client
 connection.
 .sp
 Cyrus admins who authenticate and authorize as themselves (e.g. don\(aqt
@@ -105,7 +105,7 @@ SERVICES {
 /etc/cyrus.conf
 .SH SEE ALSO
 .sp
-master(8),
+cyrmaster(8),
 imapd.conf(5),
 cyrus.conf(5)
 .SH AUTHOR
diff --git a/man/tls_prune.8 b/man/tls_prune.8
index 4129482..7f82e45 100644
--- a/man/tls_prune.8
+++ b/man/tls_prune.8
@@ -89,7 +89,7 @@ Use the specified configuration file \fIconfig\-file\fP rather than the default
 .SH SEE ALSO
 .sp
 imapd.conf(5),
-master(8)
+cyrmaster(8)
 .SH AUTHOR
 The Cyrus Team, Nic Bernstein (Onlight)
 .SH COPYRIGHT
